mmdet/apis/ote/apis/detection/openvino_task.py

```diff
@@ -104,6 +105,49 @@ def forward(self, inputs: Dict[str, np.ndarray]) -> Dict[str, np.ndarray]:
         return self.model.infer_sync(inputs)
 
 
+class OpenVINOMaskInferencer(BaseInferencer):
+    def __init__(
+        self,
+        hparams: OTEDetectionConfig,
+        label_schema: LabelSchemaEntity,
+        model_file: Union[str, bytes],
+        weight_file: Union[str, bytes, None] = None,
+        device: str = "CPU",
+        num_requests: int = 1,
+    ):
+        model_adapter = OpenvinoAdapter(
+          create_core(),
+          model_file,
+          weight_file,
+          device=device,
+          max_num_requests=num_requests)
+
+        self.configuration = {
+          **attr.asdict(
+            hparams.postprocessing,
+            filter=lambda attr, value: attr.name not in [
+              'header', 'description', 'type', 'visible_in_ui'])}
+
+        self.model = Model.create_model(
+          'maskrcnn',
+          model_adapter,
+          self.configuration,
+          preload=True)
+
+        self.converter = MaskToAnnotationConverter(label_schema)
```

Comment:
Is this converter only applicable for OpenVINO task, or can be reused to PyTorch task as well?

Reply:
not really, 1. `_add_predictions_to_dataset` in inference task also filters low score prediction, 2. `MaskToAnnotationConverter.convert_to_annotation` returns `AnnotationSceneEntity`, which then we have to extract the annotation list.